build.gradle

```diff
@@ -141,6 +141,8 @@ dependencies {
     implementation 'com.vdurmont:semver4j:3.1.0'
     implementation 'org.apache.tika:tika-core:2.9.2'
 
+    implementation 'org.apache.commons:commons-exec:1.3'
```

Comment:
Needs to be included in the shadow task to end up in release build.

Reply:
It already is. Mcef needs it too.

Reply:
Ah alright.